robottelo/ui/contentviews.py

```diff
@@ -450,6 +462,41 @@ def add_remove_errata_to_filter(self, cv_name, filter_name,
         else:
             self.click(locators['contentviews.remove_errata'])
 
+    def edit_erratum_date_range_filter(
+            self, cv_name, filter_name, errata_types=None, date_type=None,
+            start_date=None, end_date=None, open_filter=True):
+        """Edit Erratum Date Range Filter"""
+        allowed_errata_types = ['security', 'enhancement', 'bugfix']
+        allowed_date_types = ['updated', 'issued']
+        if not errata_types:
```

Comment:
hmm, it seems that `errata_types=['security']` in method signature will do the same trick